src/main/resources/application.yaml

```diff
@@ -44,3 +44,6 @@ accesstoken:
   serviceConfig:
     - serviceName: sscs
       validity: ${SAS_TOKEN_VALIDITY:300} #In seconds
+
+scan:
+  interval: ${SCAN_INTERVAL:30000}
```

Comment:
You will need to define `SCAN_INTERVAL` in terraform else code cannot be deployed.